### ISSUE TITLE
Regarding the calculation and optimization of entropy loss.

### ISSUE THREAD

**Issue by tangtaotao123** [ruoqizzz/entropy-offlineRL#2]
Thank you very much to the author for the excellent work and making the code open source. But I have a problem here: The code for calculating entropy loss is: entropy_loss = -entropy.mean() / entropy.abs().mean().detach(), and since entropy = -log_p.sum(dim=1, keepdim=True) always gives a positive result (the logarithmic probability is always negative), actually the entropy_loss is always -1 and it does not play a guiding role in policy learning. Could you explain this problem in detail? Thank you.
